server/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -139,7 +140,7 @@ public void onIndexModule(IndexModule indexModule) {
     }
 
     @Before
-    private void resetCount() {
+    public void resetCount() {
```

Comment:
oh boy :) not the first time I see this. It is still mysterious how this would not cause an error, because a before method ought to be public, but who knows. Thanks for fixing it.